testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -35,12 +35,12 @@
 from spinnaker_testing import azure_scenario_support
 
 PLATFORM_SUPPORT_CLASSES = [
-    aws_scenario_support.AwsScenarioSupport,
-    # appengine depends on google so order it after
-    google_scenario_support.GoogleScenarioSupport,
-    appengine_scenario_support.AppEngineScenarioSupport,
-    kubernetes_scenario_support.KubernetesScenarioSupport,
-    openstack_scenario_support.OpenStackScenarioSupport,
+    # aws_scenario_support.AwsScenarioSupport,
+    # # appengine depends on google so order it after
+    # google_scenario_support.GoogleScenarioSupport,
+    # appengine_scenario_support.AppEngineScenarioSupport,
+    # kubernetes_scenario_support.KubernetesScenarioSupport,
+    # openstack_scenario_support.OpenStackScenarioSupport,
     azure_scenario_support.AzureScenarioSupport
```

Comment:
are these a problem>
Why did you need to comment them out? Is there something I need to fix/change?
(and, obviously uncomment them)